synapse/config/saml2_config.py

```diff
@@ -256,6 +256,10 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
             #  remote:
             #    - url: https://our_idp/metadata.xml
 
+            # Allowed clock difference in seconds from IdP. Uncomment the below to 
+            # increase the accepted time difference from 0 to 3 seconds.
```

Comment:
```suggestion
            # Allowed clock difference in seconds between the homeserver and IdP.
            #
            # Uncomment the below to increase the accepted time difference from 0 to 3 seconds.
            #
```

I think that will meet the "style" needs that are failing?

Reply:
@clokep I've copied your proposed changes. Thank you for input, time and help.